app/client/build_runner.py

```diff
@@ -164,7 +164,7 @@ def _download_and_extract_results(self, timeout=None):
                     for chunk in response.iter_content(chunk_size):
                         file.write(chunk)
 
-                app.util.fs.extract_tar(download_filepath, delete=True)
+                app.util.fs.unzip_directory(download_filepath, delete=True)
```

Comment:
This is part of the CLI client. The type of archive it uses internally to download results doesn't *really* matter, but it's better to change it now since the plan is to eventually remove tar.gz result support.